src/native/libs/System.Security.Cryptography.Native/pal_ecc_import_export.c

```diff
@@ -592,3 +601,233 @@ EC_KEY* CryptoNative_EcKeyCreateByExplicitParameters(
 
     return ret;
 }
+
+static ECCurveType NIDToCurveType(int fieldType)
+{
+    if (fieldType == NID_X9_62_characteristic_two_field)
+        return Characteristic2;
+
+    if (fieldType == NID_X9_62_prime_field)
+        return PrimeShortWeierstrass;
+
+    return Unspecified;
+}
+
+// This is unfortunate place to put definition of this function but ECCurveType is defined in pal_ecc_import_export.h
+// and placing include in the pal_evp_pkey.h breaks the opensslshim build as it redefines the enum (enums define global names).
```

Comment:
```suggestion
```

It's a fine place.  And, even if it wasn't, that's not a comment we need in perpetuity.